src/python/ensembl/io/genomio/assembly/download.py

```diff
@@ -277,10 +270,35 @@ def retrieve_assembly_data(
 def main() -> None:
     """Module's entry-point."""
     parser = ArgumentParser(description="Download an assembly data files from INSDC or RefSeq.")
+    parser.add_argument("-v", "--verbose", action="store_true", required=False,
+        help="Verbose level logging")
+    parser.add_argument("-d", "--debug", action="store_true", required=False,
+        help="Debug level logging")
```

Comment:
Maybe we should place these after the mandatory arguments ?

Reply:
Sure I can do, only reason I put them there was I thought it was needed given they are optional flag not positional args

"The [add_argument()](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_argument) method must know whether an optional argument, like -f or --foo, or a positional argument, like a list of filenames, is expected. The first arguments passed to [add_argument()](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_argument) must therefore be either a series of flags, or a simple argument name."

Reply:
I believe that sentence refers to the first arguments passed to the function `add_argument()`, i.e. `add_argument("-f", "--foo", ...)`, and not to the parser.